region/maridia/outer.json

```diff
@@ -6225,6 +6525,89 @@
               "usableComingIn": false,
               "openEnd": 1
             }
+          ],
+          "leaveWithGMode": [
+            {
+              "leavesWithArtificialMorph": false,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Through Tube (Bottom to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,3],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "Morph",
+                    {"or":[
+                      "Gravity",
+                      {"and":[
+                        "canSuitlessMaridia",
+                        "canUseFrozenEnemies",
+                        "HiJump",
+                        "h_canCrouchJumpDownGrab"
+                      ]},
+                      {"and":[
+                        "canSuitlessMaridia",
+                        {"ammo": { "type": "Super", "count": 1 }},
+                        "canTrickyUseFrozenEnemies"
+                      ]},
+                      {"and":[
+                        "canSuitlessMaridia",
+                        "canTrickyJump",
+                        "canSpringBallJumpMidAir",
+                        "canTrickyUseFrozenEnemies"
+                      ]}
+                    ]}
+                  ],
+                  "devNote": "There are many ways to get up with Gravity, but it can be done with no other items with a gravity jump."
+                }
+              ]
+            },
+            {
+              "leavesWithArtificialMorph": true,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": true,
+                      "mode": "any"
+                    }},
+                    {"or":[
+                      "Morph",
+                      "SpringBall",
+                      {"and":[
+                        "Gravity",
+                        "h_canArtificialMorphIBJ"
+                      ]},
+                      {"and":[
+                        "Gravity",
+                        "h_canArtificialMorphBombHorizontally",
+                        {"or": [
```

Comment:
You can turn off gravity and roll to the middle to save a PB.

To PB jump the entire way with the momentum trick, you have to back up for more roll speed making it really hard.

Reply:
With the G-Mode starting position from the left door, it looks like you wont have enough momentum to do the same. You could back up, but the crabs will be headed for Samus (unless immobile is being used). I think I'll keep it only on the `R->L` for now. We should probably add a way to know if a strat is coming from immobile or not.